docs/manual/other-topics/dialect-specific-things.md

```diff
@@ -1,218 +1,236 @@
-# Dialect-Specific Things

```

Comment:
Why is there no better diff displayed by GitHub? Did you change the line endings?

Reply:
I am with VSCode under macOS and it does whatever default save is doing....

Reply:
Anything I could fix or add in lint?

Reply:
I'm not sure, I will take a look later today

Reply:
Okay, I checked it out. Apparently it was CRLF before, while we use LF (as also defined in the editorconfig file). So while you indeed changed the line ending, you changed it for the better. Thanks!